homeassistant/components/zha/core/helpers.py

```diff
@@ -68,7 +68,10 @@
     from zigpy.exceptions import DeliveryError
 
     attr_name = cluster.attributes.get(attr, [attr])[0]
-    attr_id = get_attr_id_by_name(cluster, attr_name)
+    attr_id = (
```

Comment:
Is was thinking:
```py
if isinstance(attr, str):
    attr_id = get_attr_id_by_name(cluster, attr_name)
else:
    attr_id = attr
```

Reply:
I would have thought you would have just said "remove the ternary"  ;)

Done. 

For the future, is it preferred to avoid ternary statements?

Reply:
Python's ternary statement is:
```py
a if condition else b
```
If that fits on one line, it's ok. But if it would span more lines, we prefer not to use it.